pkg/actors/actors.go

```diff
@@ -718,6 +720,32 @@ func (a *actorsRuntime) GetState(ctx context.Context, req *GetStateRequest) (*St
 	}, nil
 }
 
+func (a *actorsRuntime) DeleteState(ctx context.Context, req *DeleteStateRequest) (*runtimev1pb.DeleteActorStateResponse, error) {
```

Comment:
Please don't return a proto here. Create a specific `DeleteStateResponse` object

Also, best to avoid pointers:


```suggestion
func (a *actorsRuntime) DeleteState(ctx context.Context, req DeleteStateRequest) (DeleteStateResponse, error) {
```